wiki/fabrication/fabrication_considerations_for_3D_printing.md

```diff
@@ -16,7 +16,7 @@ It is most important to keep this in mind when you are drawing very thin walls.
 
 
 ## 2. Part Orientation
-![N|Solid](http://www.ap242.org/image/image_gallery?uuid=a7eb3359-8620-457d-9923-9d4cbec5ba5e&groupId=52520&t=1503324782232)
+![N|Solid](assets/images/part_orientation.jpeg)
```

Comment:
If you view this in "rich text" it renders correctly. However, the image description belongs in the `[]`.

Reply:
Resolved